README.md

```diff
@@ -42,15 +42,15 @@ $ cp -a build/html/ <path_to_scitools.org.uk_repo>/iris/docs/v1.12
 2. Update the `latest` symlink.
 ```
 $ cd <path_to_scitools.org.uk_repo>/iris/docs
-$ rm latest
+$ unlink latest
 $ ln -s v1.12 latest
-$ cd -
 ```
 
 3. Update available versions in `iris/docs/version_switch.js`.
 
 4. Generate the shared_assets.
 ```
+$ cd <path_to_scitools.org.uk_repo>
```

Comment:
These are instructions in the README that help some to update the docs for iris/cartopy.
When I was following the instructions, I found a couple of the instructions needed fixing